src/ImageSharp/Formats/Bmp/BmpDecoderCore.cs

```diff
@@ -492,7 +468,7 @@ private void UncompressRle4(BufferedReadStream stream, int w, Span<byte> buffer,
                         int max = cmd[1];
                         int bytesToRead = (int)(((uint)max + 1) / 2);
 
-                        Span<byte> run = bytesToRead <= 128 ? scratchBuffer.Slice(0, bytesToRead) : new byte[bytesToRead];
+                        Span<byte> run = bytesToRead <= 128 ? scratchBuffer[..bytesToRead] : new byte[bytesToRead];
```

Comment:
I am curious if there is a Analyzer to enforce this style. I know that Rider has a editoconfig setting for this

Reply:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0057

You can find most of the language style rules here: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/language-rules